nntrainer/tensor/task.h

```diff
@@ -152,14 +152,14 @@ class TaskAsync : public Task {
    *
    * @param time timeout in T
    */
-  virtual void setTimeout(int time) { timeout = T(time); }
+  virtual void setTimeout(long time) { timeout = T(time); }
 
   /**
    * @brief Get timeout
    *
    * @param time timeout in T
    */
-  virtual int getTimeout(void) const { return timeout.count(); }
+  virtual long getTimeout(void) const { return timeout.count(); }
```

Comment:
Why long? Is this coming from ```__kernel_time_t``` or ```time.h```?

Reply:
Thanks for checking that. 
I expect std::chrono::xxx::count(). (https://en.cppreference.com/w/cpp/chrono/duration)
so, I modifed to use int64_t.